src/common/com/intellij/plugins/haxe/haxelib/HaxelibProjectUpdater.java

```diff
@@ -468,11 +468,11 @@ private void syncOneModule(@NotNull final ProjectTracker tracker, @NotNull Modul
           if (file != null && file.getFileType().equals(HXMLFileType.INSTANCE)) {
             PsiFile psiFile = PsiManager.getInstance(project).findFile(file);
             if (psiFile != null) {
-              Collection<HXMLClasspath> hxmlClasspaths = PsiTreeUtil.findChildrenOfType(psiFile, HXMLClasspath.class);
-              for (HXMLClasspath hxmlClasspath : hxmlClasspaths) {
-                String classpath = hxmlClasspath.getValue();
-                haxelibExternalItems.add(new HaxelibItem(classpath, VfsUtil.pathToUrl(classpath)));
-              }
+              //Collection<HXMLClasspath> hxmlClasspaths = PsiTreeUtil.findChildrenOfType(psiFile, HXMLClasspath.class);
```

Comment:
Why comment these out for this PR?  If you want to change how the haxe libs are auto-added, (and they definitely need reworking,) I would put them into a separate PR.